src/webui/btjson.cpp

```diff
@@ -34,6 +34,8 @@
 #include <QElapsedTimer>
 #include <QVariant>
 
+#include <libtorrent/version.hpp>
```

Comment:
you need this later?

Reply:
https://github.com/qbittorrent/qBittorrent/pull/7566/files#diff-a0b4caf6b479786e86d228562b0049e0R267